frontend/src/components/pages/Home.js

```diff
@@ -1,7 +1,75 @@
+// Home.js
+import React from 'react';
+import './pages/homepagestyle.css';
+
 export function Home({ ApiStatus }) {
     return (
-        <div className="p-6">
-            <p className="mt-2">API Response: {ApiStatus}</p>
+        <div>
+            <main>
+                <section className="intro">
+                    <h2>Welcome to the Platform</h2>
+                    <p>Your one-stop destination for interactive and engaging online learning.</p>
+                    <div className="buttons">
+                        <button className="btn">Learn More</button>
+                        <button className="btn">Learn More</button>
```

Comment:
No need to have two buttons here, just one is enough.
Maybe the ``Learn More`` button could redirect to our GitHub README.md file.

Example:
```js
<a href='https://github.com/sandbox-science/online-learning-platform/blob/main/README.md' target="_blank">
    <button className="btn">Learn More</button>
</a>
```

EDIT: After more thought and based on the comment I left on line 25 (the HOME section). I think we could remove this button and have the link as I show above in the ABOUT (MISSION) box as I mentioned too. This button can then be removed.